docs/constants.md

```diff
@@ -12,6 +12,11 @@ title: Constants
 * `0` read-write
 * `1` read-only
 
+## Listable scope
+* `0` participants only
+* `1` regular users only, excluding guests
```

Comment:
Is this guest app users or guests?

Reply:
as originally clarified in the  ticket https://github.com/nextcloud/spreed/issues/1763#issuecomment-721008640 it's for "guest users from the guest app"

because for public guests (invited by token) we don't have a UI for searching for rooms or switching